collector/src/descriptor/kind/bridge_server_descriptor.rs

```diff
@@ -49,6 +73,40 @@ impl BridgeServerDescriptor {
             )
         };
 
+        let master_key = {
+            let v = take_uniq(&mut desc, "master-key-ed25519", 1)?;
+            v[0].to_owned()
+        };
+
+        let (ipv6, or_port_v6) = {
+            let v = take_opt(&mut desc, "or-address", 1)?;
+            if let Some(t) = v.map(|x| x[0].split("]:").collect::<Vec<_>>()) {
+                (
+                    Some(t[0][1..].parse().unwrap()),
+                    Some(t[1].parse().unwrap()),
+                )
+            } else {
+                (None, None)
+            }
+        };
+
+        let platform = {
+            let v = take_uniq(&mut desc, "platform", 1)?;
+            v.join(" ")
+        };
+
+        let proto = {
+            let v = take_uniq(&mut desc, "proto", 12)?;
+            let it = v.iter();
+            let res: HashMap<String, String> = it.fold(HashMap::new(), |mut res, val| {
+                let t = val.split('=').collect::<Vec<_>>();
```

Comment:
Could you use [`split_once`](https://doc.rust-lang.org/std/primitive.str.html#method.split_once). Currently if a value contains `=`, it's going to be stripped of, and if a key doesn't contain an `=`, it will panic